examples/c/ex_schema.c

```diff
@@ -201,169 +200,6 @@ main(int argc, char *argv[])
     /*! [Search in a composite index] */
     error_check(cursor->close(cursor));
 
-    /*! [Return a subset of values from the table] */
```

Comment:
This was removed due to git problems. It is part of wt-12472, not this PR.  Put back.